dkpro-jwpl-datamachine/pom.xml

```diff
@@ -44,12 +44,12 @@
     <dependency>
       <groupId>org.apache.logging.log4j</groupId>
       <artifactId>log4j-slf4j2-impl</artifactId>
-      <scope>provided</scope>
+      <scope>runtime</scope>
```

Comment:
"runtime" is a transistive scope. Loggign implementations should not be transitive dependencies of libraries. So "provided" should be correct here.

Reply:
So `optional=true` would be sufficient here. But I have no strong opinion as I am not a CLI user of those modules.

Reply:
As long as the fat-jar includes the required logger deps (with _provided_ scope set) via Maven assembly config, I'm fine with leaving this as `provided`. Will check and report back.

Reply:
I would assume that if it does not, there should be a way to add these explicitly for fat-jar inclusion only, maybe directly in the assembly or plugin configuration